packages/material-ui/src/TableRow/TableRow.js

```diff
@@ -93,6 +105,10 @@ TableRow.propTypes = {
    * Either a string to use a DOM element or a component.
    */
   component: componentPropType,
+  /**
+   * If `true`, the row will have less height.
+   */
+  dense: PropTypes.bool,
```

Comment:
This would only be needed to use `TableRow` outside of `Table`. Is this a use case?

Reply:
Would like to get rid of it. It's always problematic to have multiple sources of some internal state (e.g. copy props into state). It's probably only needed for testing and I'd rather encourage an integration test with the `Table`.

Reply:
I'll add a test and update the demo later